parachains/runtimes/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -398,6 +413,92 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Add bridge pallets (GPA)
+/// Add GRANDPA bridge pallet to track Polkadot relay chain on Kusama BridgeHub
+pub type BridgeGrandpaPolkadotInstance = pallet_bridge_grandpa::Instance1;
+impl pallet_bridge_grandpa::Config<BridgeGrandpaPolkadotInstance> for Runtime {
+	type RuntimeEvent = RuntimeEvent;
+	type BridgedChain = bp_polkadot::Polkadot;
+	type MaxFreeMandatoryHeadersPerBlock = ConstU32<4>;
+	type HeadersToKeep = RelayChainHeadersToKeep;
+	type WeightInfo = weights::pallet_bridge_grandpa::WeightInfo<Runtime>;
+}
+
+parameter_types! {
+	pub const RelayChainHeadersToKeep: u32 = 600;
+	pub const ParachainHeadsToKeep: u32 = 300;
+	/// Delay (in blocks) before registered relayer could get its stake back. It guarantees
+	/// that all pending delivery transactions are either dropped or mined and relayer is
+	/// slashed in case of misbehavior. So this value should be large enough (e.g. 24 hours).
+	pub const RelayerStakeLease: u32 = 7_200;
+	pub const PolkadotBridgeParachainPalletName: &'static str = bp_polkadot::PARAS_PALLET_NAME;
+	pub const MaxPolkadotParaHeadDataSize: u32 = bp_polkadot::MAX_NESTED_PARACHAIN_HEAD_DATA_SIZE;
+
+	// Both are just initial values, and concrete values will be set up by governance call (`set_storage`) with `initialize` bridge call as a part of cutover plan (https://github.com/paritytech/parity-bridges-common/issues/1730)
+	pub storage DeliveryRewardInBalance: u64 = 1_000_000;
+	pub storage RequiredStakeForStakeAndSlash: Balance = 1_000_000;
+	pub const RelayerStakeReserveId: [u8; 8] = *b"brdgrlrs";
+}
+
+/// Add parachain bridge pallet to track Polkadot bridge hub parachain
```

Comment:
Minor docs nit, but among the `///` docs there is inconsistency between `bridge hub` and `BridgeHub`, and many don't have periods at the end of sentences. I would tidy those up.